docs/medical-api/api-reference/patient/delete-patient.mdx

```diff
@@ -6,7 +6,7 @@ api: "DELETE /medical/v1/patient/{id}"
 
 ## Query Params
 
-<ParamField query="facilityId" type="string" required>
+<ParamField query="facilityId" type="string" optional>
```

Comment:
modified to match endpoint definition. Question of why facility ID should be provided at all here? Like if its not nessecary its kinda just confusing and superfluous no?